src/Molecules/Duplicable/Duplicable.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+
+import FlexBox from '../../Templates/FlexBox'
+
+const computeNextKey = duplicated => {
+  let nextKey = 0
+
+  duplicated.map(duplicate => {
+    const key = parseInt(duplicate.key)
+
+    if ('key' in duplicate && key >= nextKey) {
+      nextKey = key + 1
+    }
+  })
+
+  return nextKey
+}
+
+const Duplicable = ({
+  model,
+  addType,
+  removeType,
+  canBeEmpty = false,
+  instancesProps = [],
+  gap = 1,
+  mb = 1,
+  btnMarginTop = 2,
```

Comment:
We do not use `withStyles` anymore, if I remember well we now try to always use `FlexBox` or `Box` component to send them CSS properties.
I do agree it seems pretty specific, maybe should I send props like `modelProps` and `cloneBtnProps`, so we are free to send any needed props in others stories?

Reply:
I don't see why we aren't using `withStyles` since this is the Mui way, which provide a clean way to override styles. Maybe @mesteche or @monzey could hint us ?

Reply:
We also can hardcode it as you proposed to, I decided to send those values as properties so you can modify them in you stories if you need to.
But these values are the one I need in my story, so if they also are ok for your story, we can hardcode them.